packages/sanes-chrome-extension/src/logic/user/accountmanager.test.ts

```diff
@@ -0,0 +1,148 @@
+import { Algorithm, ChainId } from '@iov/bcp';
+import { HdPaths } from '@iov/core';
+
+import { AccountManager, AccountManagerChainConfig } from './accountmanager';
+import { createUserProfile } from './profile';
+
+describe('AccountManager', () => {
+  const chain1: AccountManagerChainConfig = {
```

Comment:
I understand what you are trying to do here. But I do not like in the sense it increases complexity. In the real app, are we going to handle (load - parse - use) chains in this way? I have the feeling, this is a conversion just to test something in a limited scope. This does not represent reality (in the sense of code executed by the app), right?

Reply:
What this does is breaking down a big problem into smaller problems. Those smaller problems can be looked at, solved and tested one by one. As soon as the smaller problem is solves, we can look at the bigger problem while being sure, that our small problem is solved in a solid way.

The existance of AccountManager makes it possible to handle all the accounts+chains logic without bothering with a network connection.